nixos/modules/system/activation/activation-script.nix

```diff
@@ -119,6 +119,12 @@ in
 
     system.activationScripts.stdio = ""; # obsolete
 
+    system.activationScripts.iproute2 = ''
+      cp -R ${pkgs.iproute}/etc/iproute2 /etc
```

Comment:
but does this not overwrite any modification done by the user?

Reply:
I that case, would it not be better to have config options for those files? This feels wrong. The files should either be managed by nix or not touched like that at all..

Reply:
I agree with that.

Reply:
once iproute is installed, users expect `ip route list table main` to work but if iproute can't find the file than it breaks this expection. While the file can be modified, it does have default. Maybe iproute should be patched look somewhere else than /etc/iproute2 (like in /var/run) but some sort of default should exist. To prevent overwriting user changes, the activationScript could write it only if it does'nt already exist.